src/components/Dashboard/Overview.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="dashboard-overview-container">
+    <div class="row">
+      <div class="col-4">
+        <app-event-overview></app-event-overview>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import AppEventOverview from './../Events/Overview'
```

Comment:
Why the App**Event**Overview naming?

Reply:
I had EventOverview and Dashboard Overview first. But maybe Dashboard should be one main component and be fed with minor components. So I changed the name to App**Event**Listing if we get something like "AppPollListing" in the future.